src/scripts/common.js

```diff
@@ -163,11 +164,16 @@ window.togglbutton = {
     let len;
 
     const elems = document.querySelectorAll(selector);
+    if (!elems.length) {
+      return;
+    }
+
```

Comment:
The above is the only actual change in this file, the other things were just to get it to pass lint.